### ISSUE TITLE
Add reset button to counter UI and functionality

### ISSUE THREAD

**Issue by joshtom** [joshtom/react-redux-starter-template#34]
This reset button should beable to reset the counter back to 0

**Comment by yajna-pandith** [comment]
Hi can i take up this issue :)

**Comment by joshtom** [comment]
Of course you can. I will assign you to it now

**Comment by yajna-pandith** [comment]
Hi, How to compile/debug in vs code?
What I wanted to specifically know was, 
Is there a specific debugger I should be using or just this tool called React Native Tools
 I am a newbie.

**Comment by joshtom** [comment]
No. This is react, and for this I don't think you need any debugger. After cloning the project into your machine just run **npm install** from your terminal and after that, run **npm start** to run the project. Let me know if you have any further questions.

**Comment by yajna-pandith** [comment]
Hi, Thanks it worked.
I got the functionality of the reset button right but I am facing a slight difficulty in getting a button (the SVG with the path ). For now, I have made another (less than [<]) button for which the reset functionality works. 

<img width="1787" alt="Screenshot 2020-10-18 at 8 46 54 PM" src="https://user-images.githubusercontent.com/25138248/96372251-b0d3f180-1183-11eb-90ca-31f9caf5478e.png">

This has been my progress, wanted some help in getting the SVG path right for a button.

**Comment by joshtom** [comment]
I don't think you need an svg path in this case. All you need is a button tag that responds to a click event.... And this event will be go reset the counter to 0
